shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -336,8 +318,29 @@ public void onChange(boolean selfChange, Uri uri) {
         // Tell Flutter whether touch exploration is initially active or not. Then register a listener
         // to be notified of changes in the future.
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+            touchExplorationStateChangeListener = new AccessibilityManager.TouchExplorationStateChangeListener() {
```

Comment:
Generally, I think we should avoid in-line anonymous definitions beyond anything more compicated than a click handler.  If we can't define this up above where it was then we should define it as a static inner class.  I will make this change next week.